app/src/main/java/com/nononsenseapps/feeder/ui/compose/feed/FeedScreen.kt

```diff
@@ -495,145 +509,393 @@ fun FeedScreen(
         onDelete = onDeleteFeeds,
         onEditFeed = onEditFeed,
         toolbarActions = {
-            if (viewState.currentFeedOrTag.isNotSavedArticles) {
-                PlainTooltipBox(
-                    tooltip = {
-                        Text(stringResource(id = R.string.filter_noun))
+            if (viewState.searchBarVisible) {
+                SearchBar(
+                    expanded = true,
+                    onExpandedChange = { onShowSearchBar(it) },
+                    inputField = {
+                        SearchBarDefaults.InputField(
+                            query = viewState.filter.search,
+                            onQueryChange = { filterCallback.searchFor(it) },
+                            onSearch = {
+                                onShowSearchBar(false)
```

Comment:
I'm not sure I think it's a good idea to close the search bar and leave the filter in place.

A user might make a search, press "enter", and then have no idea why all articles aren't showing much later.
```suggestion
```